### DISCUSSION TITLE
handleKeyDown event.key not working on Android phones

### DISCUSSION THREAD

**Discussion by pszafer** [JedWatson/react-select#5200] [Q&A]
Hi,
It seems on Android phones (Samsung keyboard, Google keyboard) `Enter` keycode are not populated. Browser send:
`key: "Unidentified", keyCode: 229`. If website has more than one input it just switches to next input.
Because of this I can't use Multi-select text input example. 

https://codesandbox.io/s/wmp567?module=/example.tsx

Open this site in Android Chrome browser, click to run and try to type some text. It just loose focus.
Do you have any workaround to get this working?

**Comment by tychtych** [comment]
Also working on the same issue, wondering if anyone got a workaround?
